test/tools/yulopti.cpp

```diff
@@ -190,6 +192,12 @@ class YulOpti
 				StackCompressor::run(m_dialect, obj, true, 16);
 				break;
 			}
+			case 'R':
+			{
+				ReasoningBasedSimplifier::run(context, *m_ast);
+				break;
+			}
+
```

Comment:
Extra blank

Reply:
Is there actually a reason why this is explicitly added here?

Reply:
Before I had it in the list inside Suite, so this here wasn't needed. But the `phaser` uses that list for the individuals and there were some problems with it. At that time I found it easier to remove it from the Suite list and just have it extra here for manual testing.